code/modules/research/techweb/all_nodes.dm

```diff
@@ -192,7 +192,7 @@
 	prereq_ids = list("bluespace_basic", "engineering")
 	design_ids = list("bs_rped","minerbag_holding", "bluespacebeaker", "bluespacesyringe", "phasic_scanning", "roastingstick", "ore_silo")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
-	export_price = 5000
+	export_price = 7500
```

Comment:
?????
this is under practical bluespace research and it isnt even the right var

Reply:
huh